cmd/rm.go

```diff
@@ -37,25 +34,49 @@ func rm(cmd *cobra.Command, args []string) (err error) {
 		return
 	}
 	if pathMetaData.Tag == folder {
-		return fmt.Errorf("rm: cannot remove ‘%s’: Is a directory", path)
+		if !recursive {
+			return fmt.Errorf("rm: cannot remove ‘%s’: Is a directory", path)
+		}
+		entries, err := ListFiles(path)
+		if err != nil {
+			return err
+		}
+		for _, e := range entries {
+			switch e.Tag {
+			case "folder":
+				removePath(pathmod.Join(path, e.Folder.Name), recursive)
+			case "file":
+				removePath(pathmod.Join(path, e.File.Name), recursive)
+			}
+		}
 	}
 
 	arg := files.NewDeleteArg(path)
 
 	if _, err = dbx.Delete(arg); err != nil {
 		return
 	}
+	return nil
+}
 
-	return
+func rm(cmd *cobra.Command, args []string) (err error) {
+	if len(args) != 1 {
+		return errors.New("`rm` requires a `file` or `folder` argument")
+	}
+	recursive, _ := cmd.Flags().GetBool("recursive")
+	return removePath(args[0], recursive)
 }
 
 // rmCmd represents the rm command
 var rmCmd = &cobra.Command{
 	Use:   "rm [flags] <file>",
 	Short: "Remove files",
-	RunE:  rm,
+	Example: `dbxcli rm <file>
+	dbxcli rm -r <folder>`,
```

Comment:
I thought it made sense to add that here to avoid surprising the users by mirroring the behavior of the rm command from coreutils (https://www.gnu.org/software/coreutils/manual/coreutils.html).
What do you suggest exactly?